src/Storage/Flysystem2Storage.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Kevinrob\GuzzleCache\Storage;
+
+use Kevinrob\GuzzleCache\CacheEntry;
+use League\Flysystem\Local\LocalFilesystemAdapter;
+use League\Flysystem\Filesystem;
+use League\Flysystem\FileNotFoundException;
+
+class Flysystem2Storage implements CacheStorageInterface
+{
+
+    /**
+     * @var Filesystem
+     */
+    protected $filesystem;
+
+    public function __construct(LocalFilesystemAdapter $adapter)
```

Comment:
I agree with @diego-sorribas this shouldnt be fixed to local

Reply:
Ok, I think I got it. :)
Thanks for the hints!